app/lib/common/trxHelper.js

```diff
@@ -234,15 +234,29 @@ function estimateFee(op_type, options, globalObject, data = {}) {
         op_code,
         1
     ]);
-    /* Default to transfer fees if the op is missing in globalObject */
-    if (!currentFees)
-        currentFees = globalObject.getIn([
+
```

Comment:
I guess we need to remove the above code? If the array is not continuous, for example, if fees of op 50 is not in the array, if to get fees of op 60 in such way, the fees of op 61 will be returned?

Reply:
Yes, there was a problem, I've fixed it